lib/src/widgets/utils/ds_group_card.widget.dart

```diff
@@ -194,22 +198,32 @@ class _DSGroupCardState extends State<DSGroupCard> {
       DSMessageItemModel message = widget.documents[i];
 
       List<DSMessageItemModel> groupMsgs = group['msgs'];
-
-      if (widget.compareMessages(message, groupMsgs.last)) {
-        group['msgs'].add(message);
-        group['date'] = message.date;
-        group['status'] = message.status;
-        group['displayDate'] = message.displayDate;
+      if (widget.shouldBubbleGroup) {
+        if (widget.compareMessages(message, groupMsgs.last)) {
```

Comment:
In my opinion, DSGroupCard cannot be changed for this case, DSGroupCard's function is to group,

maybe you can use DSCard for your task.